exo-add/features/exoservice-go.feature

```diff
@@ -0,0 +1,80 @@
+Feature: scaffolding an ExoService written in Go
+
+  As a Go developer adding features to an Exosphere application
+  I want to have an easy way to scaffold an empty ExoService
+  So that I don't waste time copy-and-pasting a bunch of code.
+
+  - run "exo add service <name> exoservice-go"
+    to add a new exo-service written in Go to the current application
+  - run "exo add service" to add the service interactively
+
+
+  Scenario: calling with all command line arguments
+    Given I am in the root directory of an empty application called "test app"
+    When running "exo-add service --service-role=users --service-type=users --author=test-author --template-name=exoservice-go --model=user --description=testing" in this application's directory
```

Comment:
I've just shipped a PR this morning that changes this slightly to:
```
When running "exo-add service --service-role=users --service-type=users --author=test-author --template-name=exoservice-go --model-name=user -protection-level=public --description=testing" in this application's directory
```
the key changes being:
 - model -> model-name
- adding `--protection-level=public`, could also be `private` instead